demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1125,7 +1127,7 @@ def parse_dashboard_or_report_data(path: str, data_file_json: Dict, all_layouts:
 
 def get_general_data(path: str, packs: Dict[str, Dict] = None):
     json_data = get_json(path)
-    id_ = json_data.get('id')
+    id_ = json_data.get('id', '-')
```

Comment:
same question

Reply:
to avoid bugs such as when the entity does not have the `id` field

Reply:
removed